studio/src/store/Store.js

```diff
@@ -0,0 +1,42 @@
+import { makeObservable, observable } from 'mobx';
+import { Search } from './Search.js';
+
+import { AEM } from '@adobe/mas-commons';
+import { Fragment } from './Fragment.js';
+
+let merchDataSourceCache;
+
+export class Store {
+    /**
+     * @type {Search}
+     */
+    search = new Search();
+    /**
+     * @type {import('@adobe/mas-commons').AEM}
+     */
+    aem;
+
+    /**
+     * @param {string} bucket
+     */
+    constructor(bucket) {
+        if (!bucket) throw new Error('bucket is required'); // TODO provide better error message.
+        makeObservable(this, {
+            search: observable,
+        });
+        this.aem = new AEM(bucket);
+        ({ cache: merchDataSourceCache } =
+            document.createElement('merch-datasource'));
+    }
+
+    async doSearch(props) {
+        this.search.update(props);
+        const fragments = await this.aem.sites.cf.fragments
```

Comment:
the lightweight AEM api is a mini client for https://developer.adobe.com/experience-cloud/experience-manager-apis/api/stable/sites/

Since we are operating in headless mode,  fragment is by nature the data and not HTML/WC.

I think "merch card" as term is good enough, e.g:  I'll other a merch card in MAS Studio.